### PR TITLE
Update to use Node 6.x

### PR DESCRIPTION
**Pull request by i-like-robots**

### CODE REVIEW

File: server/lib/format-tags.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-module.exports = (tags) => {
-	return tags
+module.exports = (tags) => (
+	tags
```
Comment (bjfletcher) [inline server/lib/format-tags.js]: shouldn't this return the tags?

File: server/queries/search-query.js [hunk 2]
```diff
@@ -90,11 +86,9 @@ module.exports = (params) => {
...
 													filter: {
 														bool: {
-															must: options.tagged.map(item => {
-																return {
-																	term: { 'metadata.idV1': item }
-																};
-															})
+															must: options.tagged.map(item => (
+																{ term: { 'metadata.idV1': item } }
+															))
```
Comment (bjfletcher) [inline server/queries/search-query.js]: I have to scroll quite some way to the right to see the code... 

one for the Dev Huddle - whether to limit the number of tabs on each line to, say, 6 http://eslint.org/docs/rules/max-len#tabwidth

File: server/lib/format-tags.js (hunk 1 above)
Comment (i-like-robots) [inline server/lib/format-tags.js]: `{` => `(` 😃

File: server/queries/search-query.js (hunk 2 above)
Comment (i-like-robots) [inline server/queries/search-query.js]: Yep it's a bit scary!

screen shot 2016-08-16 at 13 02 30

https://github.com/Financial-Times/next-tag-facets-api/issues/26

**Pull request merged**